### PR TITLE
Remove autograph warning for BatchNorm layers.

### PR DESCRIPTION
**Pull request by aaron276h**
Makes everything look clean when using TF 1.14.

### CODE REVIEW

File: tensorpack/models/batch_norm.py [hunk 1]
```diff
@@ -56,6 +56,15 @@ def internal_update_bn_ema(xn, batch_mean, batch_var,
         return tf.identity(xn, name='output')
 
 
+try:
+    # When BN is used as an activation, keras layers try to autograph.convert it
```
Comment (ddn0) [inline tensorpack/models/batch_norm.py]: - BN => BatchNorm?
- probably move this comment to where `@disable_autograph` is used
- probably clean up wording, e.g., "... autograph prints out too many warnings when BatchNorm is used in a keras.layer"
- if the warning has any information about how to "properly" fix it, include it here? or maybe add a TF version so folks know when to investigate removing the `@disable_autograph` decorator.

**Review by ddn0 — APPROVED** [review]

File: tensorpack/models/batch_norm.py (hunk 1 above)
Comment (aaron276h) [reply tensorpack/models/batch_norm.py]: The only info they provide about how to fix it is to "please contact the autograph team immediately".
